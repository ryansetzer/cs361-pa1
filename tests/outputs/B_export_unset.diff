--- outputs/B_export_unset.txt	2022-09-25 14:55:18.318673536 -0400
+++ expected/B_export_unset.txt	2022-09-11 14:59:21.000000000 -0400
@@ -0,0 +1,9 @@
+$ which export
+export: dukesh built-in command
+$ echo N=${NUM}
+N=
+$ export NUM=5
+$ echo N=${NUM}
+N=5
+$ quit
+
