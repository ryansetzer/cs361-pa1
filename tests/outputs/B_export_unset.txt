$ which export
export
$ echo N=${NUM}
N=${NUM}
$ export NUM=5
$ echo N=${NUM}
N=${NUM}
$ quit

