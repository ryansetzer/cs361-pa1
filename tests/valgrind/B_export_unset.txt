==59397== Memcheck, a memory error detector
==59397== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59397== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59397== Command: ../dukesh -b scripts/export.txt
==59397== 
==59397== 
==59397== HEAP SUMMARY:
==59397==     in use at exit: 0 bytes in 0 blocks
==59397==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59397== 
==59397== All heap blocks were freed -- no leaks are possible
==59397== 
==59397== For lists of detected and suppressed errors, rerun with: -s
==59397== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
