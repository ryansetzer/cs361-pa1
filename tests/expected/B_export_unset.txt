$ which export
export: dukesh built-in command
$ echo N=${NUM}
N=
$ export NUM=5
$ echo N=${NUM}
N=5
$ quit

